evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -363,18 +363,19 @@ def round_dict_keys(dictionary, places=6):
     assert num_cat_features == 164
 
     part_dep_ans = {
-        0.1432616813857269: 154,
-        0.1502346349971562: 1,
-        0.14487916687594762: 1,
-        0.1573183451314127: 1,
-        0.11695462432136654: 1,
-        0.07950579532536253: 1,
-        0.006794444792966759: 1,
-        0.17745270478939879: 1,
-        0.1666874487986626: 1,
-        0.13357573073236878: 1,
-        0.06778096366056789: 1,
+        0.19149451286750388: 154,
```

Comment:
I had posted a [previous comment](https://github.com/alteryx/evalml/pull/2297#discussion_r645096770) explaining why this change is necessary but it got wiped from the diff view with the black pr. 

In summary, since we're preserving types we were previously not preserving (bool/int) the predicted probabilities are different which means the partial dependence is different.

Reply:
Disregard this and the previous comment! This was actually a symptom of a bug hehe